iib/web/api_v1.py

```diff
@@ -90,6 +92,27 @@ def get_builds():
     return flask.jsonify(response)
 
 
+@api_v1.route('/healthcheck')
+def get_healthcheck():
+    """
+    Respond to a health check.
+
+    :rtype: str
+    """
+    # Test DB connection
+    try:
+        db.engine.execute(text("SELECT 1"))
+    except Exception:
+        return flask.jsonify({"text": "Database health check failed"})
+
+    # Ping the workers. One heartbeat from a worker is considered sufficient. 5 second timeout.
+    result = app.control.heartbeat(reply=True, limit=1, timeout=5)
```

Comment:
If all the workers are busy running tasks, will they respond? This seems like a normal scenario which shouldn't cause the health check to fail.

If you are unable to verify this, you can just remove this check.

Reply:
I found this in the docs: https://docs.celeryproject.org/en/stable/userguide/workers.html#remote-control

It seems that expecting a heartbeat message is not very reliable, so I removed it.